js/modules/k6/grpc/client.go

```diff
@@ -357,6 +367,17 @@ func resolveFileDescriptors(
 			if err = proto.Unmarshal(raw, &fdp); err != nil {
 				return nil, fmt.Errorf("can't unmarshal proto on service %q: %w", service, err)
 			}
+			fdkey := fileDescriptorLookupKey{
+				Package: *fdp.Package,
+				Name:    *fdp.Name,
+			}
+			if _, ok := fdlookup[fdkey]; ok {
```

Comment:
We can just do:

```golang
if fdlookup[fdkey] == struct{}{} {
}
```

Even better, we can do:

```golang
seen := make(map[fileDescriptorLookupKey]bool, len(services))
...
if seen[fdkey] { ... }
```

Just a suggestion, not a review request.